middleware.go

```diff
@@ -57,11 +61,14 @@ var DefaultDevStack = []rest.Middleware{
 
 	// json pretty print
 	&rest.JsonIndentMiddleware{},
+	&requestid.RequestIdMiddleware{},
 }
 
 var DefaultProdStack = []rest.Middleware{
 
 	// logging
+	&requestlog.RequestLogMiddleware{},
+	&accesslog.AccessLogMiddleware{Format: accesslog.SimpleLogFormat},
 	&rest.AccessLogJsonMiddleware{
```

Comment:
should be removed, duplicates AccessLogMiddleware, but in json format (already removed in deviceauth, adm and inventory)